src/Fetch.ml

```diff
@@ -14,6 +17,14 @@ type formData (* XMLHttpRequest *)
 type readableStream (* Streams *)
 type urlSearchParams (* URL *)
 
+module AbortController = struct
+  type t = abortController
+
+  external signal : t -> signal = "signal" [@@bs.get]
+  external abort : t -> unit = "abort" [@@bs.send.pipe: t]
```

Comment:
The `t` argument needs to be removed here. It's implied by the `t` passed to `@bs.send.pipe`.

Reply:
Okay, sorry I'm only familiar with Reason syntax and even then some of this still alludes me. So I'm assuming the line should be changed to this?
```reason
external abort : unit -> unit = "abort" [@@bs.send.pipe: t]
```

Reply:
No, it should be just `external abort : unit = "abort" [@@bs.send.pipe: t]`.

I have [a handy cheatsheet](https://bucklescript.github.io/bucklescript/Manual.html#_binding_to_method_code_bs_send_code_code_bs_send_pipe_code) you can refer to, which might help.